.github/workflows/build.yml

```diff
@@ -109,98 +107,15 @@ jobs:
           python -m pip install --upgrade pip
           make install requirements
       - name: Run the quilc & qvm Docker images
+        if: ${{ matrix.os == 'ubuntu-latest' }}
         run: |
           docker run --rm -idt -p 5000:5000 rigetti/qvm -S
           docker run --rm -idt -p 5555:5555 rigetti/quilc -R
-      - name: Run all the unit tests & generate coverage report with pytest
-        run: make test-all
+      - name: Run all the unit tests
+        run: ${{ matrix.os == 'ubuntu-latest' && 'make test-all' || 'make test' }}
```

Comment:
Is it possible to put an if condition `matrix.os == 'ubuntu-latest'` directly inside a `run` command ?
It seems it works but just double checking.

Reply:
Yeah! The syntax `A && B || C` is a ternary operator which runs `B` if `A` is true, and otherwise `C`. An example of this can be found on the [GitHub docs](https://docs.github.com/en/actions/learn-github-actions/expressions#example).

I'd prefer to use an `if`/`else` block as I think it's more readable, but currently there is [no way](https://stackoverflow.com/questions/60916931/github-action-does-the-if-have-an-else) to do an `else` using GitHub actions.

Reply:
Thanks for the explanation!